tests/testthat/test-formulas.R

```diff
@@ -8,6 +8,14 @@ ratio <- ratios_sample$ratio
 sale_price <- ratios_sample$sale_price
 assessed <- ratios_sample$assessed
 
+# Load example data from Quintos article
+mki_ki_data <- read.csv(
+  rprojroot::find_testthat_root_file("data/mki_ki_data.csv")
```

Comment:
This is the "standard" way to find files in the `testthat` directory such as data.